Modules.md

```diff
@@ -11,6 +16,13 @@ is called a **module**. A module contains:
 Sections declare their type and byte-length. Sections with unknown types are
 silently ignored.
 
+An instance contains:
+* a [linear memory](AstSemantics.md#linear-memory);
+* [global variable](AstSemantics.md#global-variables) state;
+* (when [threading](PostMVP.md#threads) is added) thread-local state;
```

Comment:
I think it would be useful to clarify this a bit. "Thread-local state" sounds a lot like TLS, and I'm not sure that's what you mean. It's implementation detail, but worker context is the type of thread state that the embedder provides, not WebAssembly IIUC.

I guess this will get hairy... Maybe leave it as "thread-specific state" to avoid the TLS ambiguity, but leave it unspecified otherwise?